app/Http/Controllers/TranslationController.php

```diff
@@ -66,6 +67,8 @@ public function createTranslation(Request $request)
 
             return response()->json($translation);
         } else {
+            Log::error('Creation of translation has failed - some elements were missed', [$word, $sign, $desc]);
```

Comment:
Måske lige definere hvad disse tre contexual messages tilhører - altså:
`['word' => $word, 'sign' => $sign, 'description' => $desc]` så vi kan se hvad disse tilhører, frem for at blive budt på 3 "random" variabler i error log?

Reply:
Kilde: [Larvel doc](https://laravel.com/docs/5.7/logging#writing-log-messages) afsnit **Contextual Information**

Reply:
Tak